rebar.config

```diff
@@ -31,8 +31,9 @@
     {observer_cli, "1.5.0"},
     {elli, "3.2.0"},
     {jsonrpc2, {git, "https://github.com/zuiderkwast/jsonrpc2-erlang.git", {branch, "master"}}},
+    {jsx, {git, "https://github.com/talentdeficit/jsx.git", {branch, "v2.8.0"}}},
```

Comment:
nb: added this since @syuan100 recommended it.

Reply:
For what purpose? Jsone already does JSON encoding/decoding? Can we reuse that?

Reply:
this branch wasn't syncing for me, and realized that core required it https://github.com/helium/blockchain-core/blob/master/src/blockchain_worker.erl#L1093

then I realized that jsx wasn't declared as a dep for core (but it's fine for miner since miner has jsx as a dep). I mean long term solution would probably be to set the dep in core, or change the decode to use jsone.

Reply:
Can we get this fixed in core instead, please?

Reply:
that seems like a better move. +1